assets/js/meetup.js

```diff
@@ -1,9 +1,11 @@
-function meetupSearch(lat, lng) {
+$(".init").on("click", meetupSearch())
```

Comment:
Can you remove this line? I moved the `meetupSearch()` call to `app.js` so that all the code for the `.init` click event is in one place